features/promotion/applying_catalog_promotions/not_reapplying_catalog_promotions_on_variant_once_its_prices_changes.feature

```diff
@@ -0,0 +1,45 @@
+@applying_catalog_promotions
+Feature: Not reapplying catalog promotions on variant once its prices changes
+    In order to have proper discounts on variants
+    As a Store Owner
+    I do not want to have discounts reapplied on variant once its prices changes if the catalog promotion criteria are not met
+
+    Background:
+        Given the store operates on a channel named "Web-US" with hostname "web-us"
+        And the store has a "T-Shirt" configurable product
+        And it is "2022-01-01" now
+        And this product has "PHP T-Shirt" variant priced at "$100.00" in "Web-US" channel
+        And there is a catalog promotion "Winter sale" available in "Web-US" channel that reduces price by "30%" and applies on "PHP T-Shirt" variant
+        And this catalog promotion operates between "2021-12-20" and "2021-12-30"
+        And there is another catalog promotion "Spring sale" available in "Web-US" channel that reduces price by "25%" and applies on "PHP T-Shirt" variant
+        And this catalog promotion operates between "2022-04-01" and "2022-05-01"
+        And there is another catalog promotion "Surprise sale" available in "Web-US" channel that reduces price by "90%" and applies on "PHP T-Shirt" variant
+        And this catalog promotion operates between "2021-01-01" and "2023-01-01"
+        And this catalog promotion is disabled
+        And I am logged in as an administrator
+
+    @api @ui
+    Scenario: Changing the price of the variant
+        When I change the price of the "PHP T-Shirt" product variant to "$50.00" in "Web-US" channel
+        Then the visitor should still see "$50.00" as the price of the "T-Shirt" product in the "Web-US" channel
+        And the visitor should still see "$100.00" as the original price of the "T-Shirt" product in the "Web-US" channel
```

Comment:
```suggestion
        Then the visitor should see "$50.00" as the price of the "T-Shirt" product in the "Web-US" channel
        And the visitor should see "$100.00" as the original price of the "T-Shirt" product in the "Web-US" channel
```

Reply:
Similar to one of my other comments, there is no catalog promotion that will be applied as neither of them meet the criteria, and original price is not set in the scenario and in the background, so where is it set? I would expect 50$ here 🤔